### PR TITLE
Additional-resources-update

### PR DESCRIPTION
**Pull request by dianaolives**

### CODE REVIEW

File: content/welcome/additional-resources.md [hunk 1]
```diff
@@ -21,6 +21,19 @@ On this community, you can:
...
+
+* A tour of the platform.
+* Getting started with {{< product-c8y-iot >}}.
+* The IoT Developer Event Series.
+* Product demos, additional resources, and more.
+
+
+#### What's new release updates
+
+In addition, to learn about what's new in {{< product-c8y-iot >}}, our [{{< product-c8y-iot >}} Youtube channel](https://www.youtube.com/@CumulocityIoT/playlists?view=50&sort=dd&shelf_id=6) also offers content which cover the new and enhanced functionality available in a specific product release.
```
Comment (rssag) [inline content/welcome/additional-resources.md:36]:
```suggestion
In addition, to learn about what's new in {{< product-c8y-iot >}}, our [{{< product-c8y-iot >}} YouTube channel](https://www.youtube.com/@CumulocityIoT/playlists?view=50&sort=dd&shelf_id=6) also offers content which cover the new and enhanced functionality available in a specific product release.
```

not sure about "in addition" following a subsection heading

File: content/welcome/additional-resources.md [hunk 2]
```diff
@@ -21,6 +21,19 @@ On this community, you can:
...
+
+Visit the  [{{< product-c8y-iot >}} Youtube channel](https://www.youtube.com/@CumulocityIoT) for more content about the platform. Here you have access to:
+
+* A tour of the platform.
+* Getting started with {{< product-c8y-iot >}}.
+* The IoT Developer Event Series.
+* Product demos, additional resources, and more.
+
+
+#### What's new release updates
```
Comment (BeateRixen) [inline content/welcome/additional-resources.md:34]: I suggest to delete this heading entirely

File: content/welcome/additional-resources.md [hunk 3]
```diff
@@ -21,6 +21,19 @@ On this community, you can:
 * Use the online discussion forums, moderated by {{< company-sag >}} professionals, to ask questions, discuss best practices, and learn how other customers are using {{< company-sag >}} technology.
 * Link to external websites that discuss open standards and web technology.
 
+### {{< product-c8y-iot >}} Youtube channel
```
Comment (BeateRixen) [inline content/welcome/additional-resources.md:24]:
```suggestion
### Cumulocity IoT YouTube channel
```
We can´t used variables in headings for technical reasons.

File: content/welcome/additional-resources.md [hunk 4]
```diff
@@ -21,6 +21,16 @@ On this community, you can:
...
+### Cumulocity IoT YouTube channel
+
+Visit the [{{< product-c8y-iot >}} YouTube channel](https://www.youtube.com/@CumulocityIoT) for more content about the platform. Here you have access to:
+
+* A tour of the platform.
+* Getting started with {{< product-c8y-iot >}}.
+* The IoT Developer Event Series.
+* Product demos, additional resources, and more.
+
+To learn about [what's new in {{< product-c8y-iot >}}](https://www.youtube.com/@CumulocityIoT/playlists?view=50&sort=dd&shelf_id=6), our YouTube channel also offers content which cover the new and enhanced functionality available in a specific product release.
```
Comment (BeateRixen) [inline content/welcome/additional-resources.md:33]:
```suggestion
To learn about [what's new in {{< product-c8y-iot >}}](https://www.youtube.com/@CumulocityIoT/playlists?view=50&sort=dd&shelf_id=6), our YouTube channel also offers content which covers the new and enhanced functionality available in a specific product release.
```

**Pull request merged**